api-server/migrations/20220320153148-add-fields-to-current_date_times-view.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.query(`
+    CREATE OR REPLACE VIEW current_date_times_jsonb AS
```

Comment:
Do we still need this first migration? Aren't we dropping this in the next migration?

Reply:
good call, deleted 👍🏼